app/addons/auth/base.js

```diff
@@ -97,6 +97,9 @@ function (app, FauxtonAPI, Auth) {
 
     var authDenied = function () {
       var url = window.location.hash.replace('#', '');
+      if (! /login?urlback=/.test(url)) {
```

Comment:
Surprised this works... `?` in a reg exp means the previous char is optional, so it should only match `loginurlback=` or `logiurlback`. Mind verifying?